src/Nethermind/Nethermind.Merge.Plugin/Synchronization/BeaconPivot.cs

```diff
@@ -75,7 +75,6 @@ public class BeaconPivot : IBeaconPivot
 
         public long PivotDestinationNumber => CurrentBeaconPivot is null
```

Comment:
PivotDestinationNumber is used only for logging? Do we have correct logging?

Reply:
it is not 100% accurate as the destination number moves as blocks gets processed from genesis in archive sync. However it is good enough for the sync reporter